### DISCUSSION TITLE
correctly dispose useVideoTexture()

### DISCUSSION THREAD

**Discussion by hatsumatsu** [pmndrs/drei#1586] [Q&A]
Hey! I'm currently looking into using the `useVideoTexture()` hook to play some dynamic video content. Is there any need to manually cleanup after unmounting the component containing the video? In my use case they will be quite a number of video components mounted and unmounted over time. 

I know that there is `useLoader.clear( mediaUrl )` but how about disposing the `<video>` element created behind the scenes? Or will this be garbage collected by default?

**Comment by fromwhite** [comment]
same issue

**Comment by SlawomirMazgaj** [comment]
+1 the question. I'm having a video and I can still hear the audio in the background, after whole ThreeJS scene is removed.
